openpype/hosts/blender/api/lib.py

```diff
@@ -36,9 +33,10 @@ def register_module_call(mod):
             except:
                 traceback.print_exc()
         else:
-            print("\nWarning! '%s' has no register function, "
-                  "this is now a requirement for registerable scripts" %
-                  mod.__file__)
+            print(
+                "\nWarning! '%s' has no register function, "
```

Comment:
Please try to use Python 3 fast strings all over your code: 
```suggestion
                f"\nWarning! '{mod.__file__}' has no register function, "
```